docs/topics/auth/default.txt

```diff
@@ -1631,6 +1631,23 @@ provides several built-in forms located in :mod:`django.contrib.auth.forms`:
     about :ref:`using the built-in authentication forms with custom user models
     <custom-users-and-the-built-in-auth-forms>`.
 
+.. class:: AdminUserCreationForm
+
+    .. versionadded:: 5.1
+
+    A form used in the admin interface to create a new user. Inherits from
+    :class:`UserCreationForm` and it has four
+    fields: ``username`` (from the user model), ``password1``, ``password2``,
+    and ``usable_password`` (the latter is enabled by default).
```

Comment:
Maybe
```suggestion
    :class:`UserCreationForm`.

    It has an additional field ``usable_password``, which is enabled by default.
```

Reply:
Sounds good, applying!

Reply:
I also changed `versionadded` to be `5.1.1` following Carlton's suggestion.